src/navigation/odom_transformer/odom_transformer/node_odom_transformer.py

```diff
@@ -0,0 +1,80 @@
+from math import cos, pi, radians, sin
+
+from transforms3d.euler import euler2quat, quat2euler
+
+import rclpy
+from rclpy.node import Node
+
+from geometry_msgs.msg import Quaternion
+from nav_msgs.msg import Odometry
+
+
+def yaw(quat: Quaternion):
+    # Convert quaternion to euler angles
+    return quat2euler([quat.w, quat.x, quat.y, quat.z])[2]  # yaw
+
+
+def orientation(x, y, z):
```

Comment:
Potentially (in line with previous comment) comment that it converts eulerian anglers to quaternions?